rcl/test/rcl/test_init.cpp

```diff
@@ -131,6 +133,31 @@ TEST_F(CLASSNAME(TestRCLFixture, RMW_IMPLEMENTATION), test_rcl_init_and_shutdown
   EXPECT_EQ(RCL_RET_INVALID_ARGUMENT, ret);
   rcl_reset_error();
   ASSERT_FALSE(rcl_context_is_valid(&context));
+  // If an invalid ROS arg is given, init should fail.
```

Comment:
Can this unit test be broken up into smaller tests? It's doing a lot. If not, it would be nice for it to include some whitespace so it's easier to see the stages of the test.

Reply:
That's a fair point. See https://github.com/ros2/rcl/pull/684/commits/4d8478c399d6314ae8e31120a62dc3655d3831b8.